modules/installation-configuration-parameters.adoc

```diff
@@ -766,6 +766,19 @@ ifdef::rhv[]
 |`platform.ovirt.ingress_vip`
 |Required. An IP address on the machine network that will be assigned to the Ingress virtual IP (VIP).
 |String. Example: `10.46.8.232`
+
+|`platform.ovirt.affinityGroups.description`
```

Comment:
Add 


| `platform.ovirt.affinityGroups`
| Required. A list of affinity groups that should be created during the installation process.
| List of objects. See below.